subscriptions/views/widget_library.py

```diff
@@ -0,0 +1,25 @@
+from django.http import JsonResponse
+from rest_framework.decorators import api_view
+
+from associations.models import Library
+
+
+@api_view(["GET"])
+def widget_library_view(request, library_id):
+    """Displays the most borrowed loanables if they are available"""
+
+    marketplace = Library.objects.get(pk=library_id)
+
+    response = {}
+
+    suggested_loanables = marketplace.loanables.filter(
+        real_return_date__is_null=False
+    ).all()
+    suggested_loanables = sorted(
+        suggested_loanables,
+        key=lambda loanable: loanable.transactions.filter(buyer=request.user).count(),
```

Comment:
I think you confused `transactions` (for `Product` and with a `buyer`) and `loan` (for `Loanable` and with an `user`).

Reply:
Can't we do this sort with an `order_by` clause?

Reply:
Maybe with an annotation, as we are using an aggregation function (count)